cmd/root.go

```diff
@@ -59,6 +63,13 @@ var RootCmd = &cobra.Command{
 func init() {
 	RootCmd.PersistentFlags().IntVarP(&numRetries, "retries", "r", DefaultNumRetries, "For SSM, the number of retries we'll make before giving up")
 	RootCmd.PersistentFlags().BoolVarP(&verbose, "verbose", "", false, "Print more information to STDOUT")
+	RootCmd.PersistentFlags().StringVarP(&backendFlag, "backend", "b", "ssm",
+		`Backend to use; AKA $CHAMBER_SECRET_BACKEND
+	null: no-op
+	ssm: SSM Parameter Store
+	s3: S3; requires --backend-s3-bucket`,
+	)
+	RootCmd.PersistentFlags().StringVarP(&backendS3BucketFlag, "backend-s3-bucket", "", "", "bucket for S3 backend; AKA $CHAMBER_S3_BUCKET")
```

Comment:
One thing that I don't like is that the user can do something invalid like `chamber -b s3 --backend-s3-bucket=mybucket` and it's not an error. Validating that only the correct flags are set in the correct backend modes sounds like a chore, especially when you factor in the env vars. Well, it's probably not so bad when we only have one backend-specific flag.